<ui version="4.0" >
 <class>Qt_launchpadClass</class>
 <widget class="QMainWindow" name="Qt_launchpadClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>410</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>410</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Qt Launchpad</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QDockWidget" name="statusDockWidget" >
   <property name="minimumSize" >
    <size>
     <width>410</width>
     <height>0</height>
    </size>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Status</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="statusDockWidgetContents" >
    <property name="maximumSize" >
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Quota</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Kbyte_loadbar" name="totalQuotaProgressBar" >
       <property name="minimumSize" >
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="value" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="launcherDockWidget" >
   <property name="minimumSize" >
    <size>
     <width>410</width>
     <height>0</height>
    </size>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Launcher</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="launcherDockWidgetContents" />
  </widget>
  <widget class="QDockWidget" name="childrenDockWidget" >
   <property name="minimumSize" >
    <size>
     <width>410</width>
     <height>0</height>
    </size>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Children</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="childrenDockWidgetContents" />
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kbyte_loadbar</class>
   <extends>QProgressBar</extends>
   <header>kbyte_loadbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
